src/Tizen.Uix.VoiceControlManager/Tizen.Uix.VoiceControlManager/VoiceControlManagerClient.cs

```diff
@@ -0,0 +1,2105 @@
+        /// </privilege>
+        /// <privlevel>
+        /// public
+        /// </privlevel>
+        /// <feature>
+        /// http://tizen.org/feature/speech.control
+        /// http://tizen.org/feature/microphone
+        /// </feature>
+        /// <exception cref="InvalidOperationException">This exception can be due to invalid state.</exception>
+        /// <pre>
+        /// The State should be Ready
+        /// </pre>
+        public static void UnsetCommandList()
+        {
+            ErrorCode error = VcMgrUnsetCommandList();
+            if (error != ErrorCode.None)
+            {
+                Log.Error(LogTag, "UnsetCommandList Failed with error " + error);
+                throw ExceptionFactory.CreateException(error);
+            }
+        }
+
+        /// <summary>
+        /// Sets commands from file.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// <privilege>
+        /// http://tizen.org/privilege/voicecontrol.manager
+        /// </privilege>
+        /// <privlevel>
+        /// public
+        /// </privlevel>
+        /// <feature>
+        /// http://tizen.org/feature/speech.control
+        /// http://tizen.org/feature/microphone
+        /// </feature>
+        /// <param name="path">File Path</param>
+        /// <param name="type">Command type</param>
+        /// <exception cref="InvalidOperationException">This exception can be due to an invalid state.</exception>
+        /// <exception cref="InvalidOperationException">This exception can be due to operation failed.</exception>
+        /// <exception cref="ArgumentException">This exception can be due to an invalid parameter.</exception>
+        /// <pre>
+        /// The State should be Ready
+        /// </pre>
+        public static void SetCommandListFromFile(string path, CommandType type)
+        {
+            ErrorCode error = VcMgrSetCommandListFromFile(path, type);
+            if (error != ErrorCode.None)
+            {
+                Log.Error(LogTag, "SetCommandListFromFile Failed with error " + error);
+                throw ExceptionFactory.CreateException(error);
+            }
+
+        }
+
+        /// <summary>
+        /// Sets background commands of preloaded app from file.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// <privilege>
+        /// http://tizen.org/privilege/voicecontrol.manager
+        /// </privilege>
+        /// <privlevel>
+        /// public
+        /// </privlevel>
+        /// <feature>
+        /// http://tizen.org/feature/speech.control
+        /// http://tizen.org/feature/microphone
+        /// </feature>
+        /// <param name="filePath">File Path</param>
+        /// <exception cref="InvalidOperationException">This exception can be due to an invalid state.</exception>
+        /// <exception cref="InvalidOperationException">This exception can be due to operation failed.</exception>
```

Comment:
If this exception can be thrown in both cases, please write it as a single instance of `<exception cref="InvalidOperationException">`.